spec/views/accounts/show.html.erb_spec.rb

```diff
@@ -85,6 +85,15 @@
     end
   end
 
+  context 'when current user has ipp pending profile deactivated for password reset' do
+    let(:user) { create(:user, :with_pending_in_person_enrollment) }
```

Comment:
We should probably include similar contexts for `AccountShowPresenter#pending_ipp?` and `AccountShowPresenter#pending_gpo?` method specs.

Reply:
You meant pending_idv correct?
I have added for that and gpo

Reply:
What I mean is that we should add specs for [`#pending_gpo?`](https://github.com/18F/identity-idp/blob/b6e7326b124599bec96499b20b81f243b00527db/spec/presenters/account_show_presenter_spec.rb#L162-L172) and [`#pending_ipp`](https://github.com/18F/identity-idp/blob/b6e7326b124599bec96499b20b81f243b00527db/spec/presenters/account_show_presenter_spec.rb#L150-L160) for the presenter class to assert that those methods return `false` if the pending profile is deactivated due to password reset.